tests/_test_mscolab/test_models.py

```diff
@@ -55,3 +71,83 @@ def test_verify_password(self):
         user = User(self.userdata[0], self.userdata[1], self.userdata[2])
         assert user.verify_password("fail") is False
         assert user.verify_password(self.userdata[2]) is True
+
+
+class Test_Permission:
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        self.u_id = 1
+        self.op_id = 1
+        self.access_level = "admin"
+
+    def test_permission_creation(self):
+        permission = Permission(self.u_id, self.op_id, self.access_level)
+
+        assert permission.u_id == self.u_id
+        assert permission.op_id == self.op_id
+        assert permission.access_level == self.access_level
+
+
```

Comment:
please add also a test for
 test_permission_repr_values

maybe you can use @pytest.mark.parametrize and use all other than admin